app/controllers/post_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostController < ApplicationController
   def show
     @post = Subscriber.find(show_params)
-    @opcount = Subscriber.where('author = ?', @post.author).count
+    @author_posts = Subscriber.where('author = ?', @post.author).count
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.